test/unittests/skills/test_utterance_intents.py

```diff
@@ -0,0 +1,102 @@
+import unittest
+from mycroft.skills.intent_services.padatious_service import PadatiousService, FallbackIntentContainer
```

Comment:
At some point, this should patch the service to test padatious and padaos independently

Reply:
the regex config flag already does this, both are being tested

Reply:
Ah, missed that in the `get_service` call